src/markdown-pages/docs/first-steps/installing.md

```diff
@@ -23,7 +23,7 @@ If you do not already have an Eclipse installation, you can <a href="https://www
 
 If you already have a version of Eclipse installed, it should be at the version codenamed Oxygen (released in June 2017) or later.
 
-### A 3270 terminal emulator
+### (Optional) A 3270 terminal emulator
 
```

Comment:
How about (and advise me if this is shocking grammar) 
Although you do not need a 3270 emulator to run a galasa test (even if it tests a 3270 application) you can use one to explore Galasa Simbank, a simulated version of an application that helps you get acquainted with Galasa before connecting to a real mainframe to run your own tests